### DISCUSSION TITLE
Create issue in a filtered board regarding status

### DISCUSSION THREAD

**Discussion by Ph0tonic** [community/community#6469] [Projects and Issues] [answered]
Hello,
When you create an issue in a filtered board regarding status then the issue disappears and you have to switch to the tab where all issues appear to be able to change the status so that it appears in the right tab.

One solution to which I can think of would be to have the possibility to assign some field when an issue is created in some board. For example to automatically assign a status when created in a specific board. Another solution would be to leave the issue display and allow setting up the other fields until you leave this issue.

I search for this issue but haven't found it, hoping not having created a dupplicate.

**Comment by azenMatt** [comment, marked as answer]
We are working on a solution to this at the moment 😄

**Reply by Ph0tonic** [reply]
Awesome, thanks !

**Reply by clarkevans** [reply]
This is basic functionality I hope it's a high priority.  I can "fake" hierarchical targets by making one tab per product, and having a filter for the corresponding product on a field.  However, user interface makes it really hard since I cannot add new items on that tab.

**Reply by azenMatt** [reply]
@clarkevans - can you share the filters where this isn't working for you? We released an update for this, but are still tracking down one or two edge cases

**Reply by superrm11** [reply]
I'm having the same issue - I have a project board with a different tab for each repo of the project and every time I try to create a new draft issue it kicks back "The new item is hidden by this view's filters." and disappears.

**Reply by PrettyCosmic** [reply]
I am having the same issue too -  Create new issue only results in "The new item is hidden by this view's filters." and disappears. I want to create my "To do" in a table.

**Reply by azenMatt** [reply]
What are the filters? If it is not possible for the system to apply them then it won't show up. 

e.g if the filter `is:pr` then an issue will never show up

**Reply by superrm11** [reply]
The filter is based on the repo - when you first create a card, it's a draft and not associated with any repo. The issue is if you have a repo filter on the board, creating a new card will cause it to be hidden immediately. You then have to dig for the draft in a tab without filters, which can be unintuitive and time consuming depending on how big the board is. 

This could be solved by automatically converting a newly created card to an issue associated with the repo chosen in the filters (to avoid it being a draft at all), or by creating an "isdraft" filter that could be applied alongside the existing repo filter (eg "isdraft OR repo:xyz)

**Reply by azenMatt** [reply]
Thank you for the context. 

We are looking at changing the default to creating an issue rather than a draft, so something like this (but swapped around - so issue is `enter` and draft is `cmd+enter`) 

![image](https://github.com/community/community/assets/7584089/db3fc57e-47fa-4163-a9ac-f03673471631)


What are your initial thoughts on that - would this be a valuable change to how you use projects?

**Reply by superrm11** [reply]
This would be very helpful. 

How would this work for tabs with multiple repo filters (repo:"abc","xyz") ? Are you able to choose which repo the issue is created in with the drop-down menu?

**Reply by azenMatt** [reply]
Yes - once you are in the 'create' modal you will have the option to pick a repo. We will default to the most used repo in the project
